vrs/vrs-annotation-batch-wrapper.py

```diff
@@ -63,62 +67,90 @@ def init_job_with_gcloud(
 
 def main(args):
 
-    # Working bucket definition
-    working_bucket = args.working_bucket 
+    hl.init(backend="batch")
+
+    working_bucket = args.working_bucket
 
-    # prefix to create custom named versions of outputs 
-    prefix = args.prefix + args.version 
+    # Prefix to create custom named versions of outputs
+    prefix = args.prefix + args.version
 
     # Input paths (fixed) - note that the Hail Tables are altered outside of the script and may partition oddly
     input_paths_dict = {
-        "3.1.2": public_release("genomes").path ,
-        "test_v3_1k": "gs://gnomad-vrs/ht-inputs/ht-1k-TESTING-ONLY.ht",
-        "test_v3_10k": "gs://gnomad-vrs/ht-inputs/ht-10k-TESTING-ONLY.ht",
-        "test_v3_100k": "gs://gnomad-vrs/ht-inputs/ht-100k-TESTING-ONLY.ht",
+        "v3.1.2": public_release("genomes").path,
+        "test_v3.1.2": public_release("genomes").path,
+        "test_v3_1k": "gs://gnomad-vrs-io-finals/ht-inputs/ht-1k-TESTING-ONLY-repartition-10p.ht",
+        "test_v3_10k": "gs://gnomad-vrs-io-finals/ht-inputs/ht-10k-TESTING-ONLY-repartition-50p.ht",
+        "test_v3_100k": "gs://gnomad-vrs-io-finals/ht-inputs/ht-100k-TESTING-ONLY-repartition-100p.ht",  # updated with repartition
     }
 
     output_paths_dict = {
-        "3.1.2": v3_vrs_annotations.path,
-        "test_v3_1k": f"gs://gnomad-vrs/vrs-temp/outputs/{prefix}-Full-ht-1k-output-full-annotated.ht",
-        "test_v3_10k": f"gs://gnomad-vrs/vrs-temp/outputs/{prefix}-Full-ht-10k-output-full-annotated.ht",
-        "test_v3_100k": f"gs://gnomad-vrs/vrs-temp/outputs/{prefix}-Full-ht-100k-output-full-annotated.ht",
+        "v3.1.2": v3_vrs_annotations.path,
+        "test_v3.1.2": f"gs://gnomad-vrs-io-finals/ht-outputs/{prefix}-Full-ht-release-output.ht",
+        "test_v3_1k": f"gs://gnomad-vrs-io-finals/ht-outputs/{prefix}-Full-ht-1k-output.ht",
+        "test_v3_10k": f"gs://gnomad-vrs-io-finals/ht-outputs/{prefix}-Full-ht-10k-output.ht",
+        "test_v3_100k": f"gs://gnomad-vrs-io-finals/ht-outputs/{prefix}-Full-ht-100k-output.ht",
     }
 
-    # Reads in Hail Table, partitions, and exports to sharded VCF within the folder to be mounted
+    check_resource_existence(
+        output_step_resources={
+            "vrs-annotation-batch-wrapper.py": [
+                f"gs://{working_bucket}/vrs-temp/outputs/VRS-{prefix}",
+                output_paths_dict[args.version],
+            ]
+        },
+        overwrite=args.overwrite,
+    )
+
+    # Read in Hail Table, partition, and export to sharded VCF within the folder to be mounted
     ht_original = hl.read_table(input_paths_dict[args.version])
 
-    # Option to downsample for testing, if you want to test on v3.1.2 but not all of it
-    if args.downsample < 1.00 and args.version == "3.1.2":
+    # Option to downsample for testing, if you want to annotate part of a Hail Table but not all of it
+    if args.downsample and args.downsample < 1.00:
```

Comment:
can just change to 'if  args.downsample < 1.00' and keep the args default as 1

Reply:
will do!